src/main/startup.sql

```diff
@@ -0,0 +1,14 @@
+-- File contains DDL statements that should be run on startup.
+-- Each DDL statement must be on its own line.
+-- A line that starts with '--' is considered to be a comment.
+
+CREATE TABLE noisepage_forecast_clusters(iteration INT, cluster_id INT, query_id INT, db_id INT);
+CREATE TABLE noisepage_forecast_forecasts(iteration INT, query_id INT, interval INT, rate REAL);
+CREATE TABLE noisepage_forecast_frequencies(iteration INT, query_id INT, interval INT, seen REAL);
```

Comment:
Seems that `noisepage_forecast_clusters` and `noisepage_forecast_forecasts` are predictions and `noisepage_forecast_frequencies` is history? Could you document this a bit?